pkg/netpol/connlist/exposed_pods.go

```diff
@@ -0,0 +1,27 @@
+package connlist
+
+import "github.com/np-guard/netpol-analyzer/pkg/netpol/internal/common"
+
+// ExposedPeer captures potential ingress and egress connections data for an exposed Peer
+type ExposedPeer interface {
+	// ExposedPeer is a peer for which the analysis found some potential exposure info
+	ExposedPeer() Peer
+	// IngressExposure list of the potential Ingress connections to the ExposedPeer
+	IngressExposure() []XgressExposureData
+	// EgressExposure list of the potential Egress connections from the ExposedPeer
+	EgressExposure() []XgressExposureData
+}
+
+// XgressExposureData data of potential connectivity for an existing peer to/from a representative peer
+type XgressExposureData interface {
+	// IsProtectedByNetpols Is the exposed peer protected by any xgress netpol
```

Comment:
```suggestion
	// IsProtectedByNetpols indicates if the exposed peer is protected by any netpol on Ingress/Egress 
```
change documentation so that the first word is the field name, but also is the first word in the sentence (not a separate sentence that starts right after that) .

Reply:
done